src/utils/updateContainerApp/ContainerAppUpdateStep.ts

```diff
@@ -4,21 +4,21 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { GenericTreeItem, nonNullProp } from "@microsoft/vscode-azext-utils";
-import { type Progress } from "vscode";
+import type { Progress } from "vscode";
+import type { ImageSourceBaseContext } from "../../commands/image/imageSource/ImageSourceBaseContext";
+import { getContainerNameForImage } from "../../commands/image/imageSource/containerRegistry/getContainerNameForImage";
 import { activityFailContext, activityFailIcon, activitySuccessContext, activitySuccessIcon } from "../../constants";
 import { ext } from "../../extensionVariables";
 import { ContainerAppModel, getContainerEnvelopeWithSecrets } from "../../tree/ContainerAppItem";
-import { ExecuteActivityOutput, ExecuteActivityOutputStepBase } from "../../utils/activity/ExecuteActivityOutputStepBase";
-import { createActivityChildContext } from "../../utils/activity/activityUtils";
-import { localize } from "../../utils/localize";
-import { updateContainerApp } from "../../utils/updateContainerApp";
-import type { IDeployImageContext } from "./deployImage";
-import { getContainerNameForImage } from "./imageSource/containerRegistry/getContainerNameForImage";
+import { ExecuteActivityOutput, ExecuteActivityOutputStepBase } from "../activity/ExecuteActivityOutputStepBase";
+import { createActivityChildContext } from "../activity/activityUtils";
+import { localize } from "../localize";
+import { updateContainerApp } from "./updateContainerApp";
```

Comment:
Rather than importing this directly, shouldn't each wizard just include the execute step? It seems kind of weird to have the step, but then directly call it like this.

Reply:
Now that I'm looking at it more, `ContainerAppUpdateStep` probably belongs in the `imageSource` folder since it's pretty heavily tied to those context values.  The other two files in `updateContainerApp` I'm not sure of... kind of thinking maybe they should still sit in utils IDK yet

Reply:
Yeah, there's a bit of stuff to unpack here that I'll reach out to discuss with you.  Some topics to cover:
- Renaming `ContainerAppUpdateStep`
- Moving `ContainerAppUpdateStep` to `imageSouce/`
- Potentially keeping the renaming two files in the `utils` folder

Reply:
Summary of actions:
- Moved `ContainerAppUpdateStep` to `imageSource/`.
- Moved `updateContainerApp` and `ContainerAppOverwriteConfirmStep` to `commands/`